ArcadiaParties.CQRS/Handlers/GetAllUsersHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using ArcadiaParties.Data.Abstractions.DTOs;
+using ArcadiaParties.Data.Abstractions.Repositories;
+using MediatR;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.CQRS.Handlers
+{
+    class GetAllUsersHandler : IRequestHandler<GetAllUsersQuery, IEnumerable<UserDTO>>
```

Comment:
It is more commonly (at least in Arcadia) to not omit visibility modifier for .NET code.

Reply:
I've made classes public

Reply:
Modifiers have been written